eng/Versions.props

```diff
@@ -75,5 +75,11 @@
     <MicrosoftSourceLinkVersion>1.0.0-beta2-19554-01</MicrosoftSourceLinkVersion>
     <MicrosoftDotNetSwaggerGeneratorMSBuildVersion>1.0.0-beta.20567.6</MicrosoftDotNetSwaggerGeneratorMSBuildVersion>
     <XliffTasksVersion>1.0.0-beta.20055.1</XliffTasksVersion>
+    
+    <!-- Roslyn -->
+    <!-- WARNING: Depending on Roslyn in code that also supports unmanaged scenarios (e.g. C++,
```

Comment:
Why do we do that then?

Reply:
We ended up not referencing Roslyn, but I moved the variable anyway because we already have the code for source based discovery (in TestWindow), and this is the version used there.